onnxruntime/core/providers/cuda/cuda_execution_provider.cc

```diff
@@ -177,6 +177,10 @@ CUDAExecutionProvider::CUDAExecutionProvider(const CUDAExecutionProviderInfo& in
     }
   }
 
+#if defined(CUDA_VERSION) && CUDA_VERSION >= 10000
+  graph_.SetStream(stream_);
```

Comment:
Nit: Should we always set the stream for the Graph instance ror only if the CUDA EP is configured to use cuda graphs ?

Reply:
Updated it: the stream is set inside `OnRunStart()` right before starting cuda graph capturing.